moonraker/components/paneldue.py

```diff
@@ -425,7 +431,11 @@ def process_line(self, line: str) -> None:
         # Prepare GCodes that require special handling
         if cmd in self.special_gcodes:
             sgc_func = self.special_gcodes[cmd]
-            script = sgc_func(parts[1:])
+
+            if cmd == "M98":
+                script = sgc_func(parts)
+            else:
+                script = sgc_func(parts[1:])
```

Comment:
This doesn't appear necessary.  In the M98 handler doesn't need `parts[0]`, as that is just the command itself.